### PR TITLE
update plugin

### PR DESCRIPTION
**Pull request by eray-felek-sonarsource**

### CODE REVIEW

**Comment by eray-felek-sonarsource** [comment]
@damien-urruty-sonarsource

File: src/main/kotlin/com/github/erayfeleksonarsource/fakesonarlintplugin/services/NotificationService.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.github.erayfeleksonarsource.fakesonarlintplugin.services
+
+import com.intellij.notification.Notification
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import org.slf4j.LoggerFactory
+
+class NotificationService : AnAction() {
```
Comment (damien-urruty-sonarsource) [inline src/main/kotlin/com/github/erayfeleksonarsource/fakesonarlintplugin/services/NotificationService.kt]: You will learn it very soon but Services are a special concept in the IntelliJ platform, so I would take it out of the name.
We have some conventions around action class names:
* end with `Action`
* start with a verb

The idea is to describe the intent of your action. Here it would be something like `NotifyAction`.

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -1,14 +1,34 @@
 <!-- Plugin Configuration File. Read more: https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html -->
 <idea-plugin>
-    <id>com.github.erayfeleksonarsource.fakesonarlintplugin</id>
-    <name>FakeSonarlintPlugin Template</name>
+    <id>com.github.erayfeleksonarsource.fakesonarlintplugin-updated</id>
```
Comment (damien-urruty-sonarsource) [inline src/main/resources/META-INF/plugin.xml]: Is this change required ?

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
+        </action>
+    </actions>
+
+    <extensions defaultExtensionNs="com.intellij">
+        <notificationGroup id="Custom Notification Group"
+                           displayType="BALLOON"/>
+    </extensions>
+
     <extensions defaultExtensionNs="com.intellij">
+        <notificationGroup id="MainToolBar"
```
Comment (damien-urruty-sonarsource) [inline src/main/resources/META-INF/plugin.xml]: Do you need this group ? I think you only use the first one

File: README.md [hunk 4]
```diff
@@ -17,7 +17,7 @@
 - [ ] Click the <kbd>Watch</kbd> button on the top of the [IntelliJ Platform Plugin Template][template] to be notified about releases containing new features and fixes.
 
 <!-- Plugin description -->
-This Fancy IntelliJ Platform Plugin is going to be your implementation of the brilliant ideas that you have.
+This Fancy IntelliJ Platform Plugin is going to be your implementation of the brilliant ideas that you have updated.
```
Comment (damien-urruty-sonarsource) [inline README.md]: Is this needed ?

**Review by damien-urruty-sonarsource — APPROVED** [review]
A comment about naming, the rest is minor, apart from that LGTM :+1: 
Congrats for your first PR!

File: README.md (hunk 4 above)
Comment (eray-felek-sonarsource) [reply README.md]: In the first part we were asked to update the description, I rolled back the changes

File: src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (eray-felek-sonarsource) [reply src/main/resources/META-INF/plugin.xml]: In the first part we were asked to customize the id

**Comment by eray-felek-sonarsource** [comment]
> A comment about naming, the rest is minor, apart from that LGTM 👍 Congrats for your first PR!

Thank you Damien, I have made the changes and pushed to my branch
